static/gsApp/views/amCheckout/checkoutSuccess.tsx

```diff
@@ -0,0 +1,671 @@
+        "We've processed your payment and updated your subscription. Your plan will renew on [date].",
+        {date: renewalDate}
+      )
+    : effectiveToday
+      ? t("We've updated your subscription.")
+      : tct('No charges today. Your subscription will update on [date].', {
+          date: effectiveDate,
+        });
+
+  return (
+    <Flex
+      padding="2xl"
+      maxWidth="1440px"
+      align="center"
+      justify="between"
+      gap="3xl"
+      direction={{sm: 'column', md: 'row'}}
+    >
+      <Flex direction="column" align={{sm: 'center', md: 'start'}} maxWidth="500px">
+        <Title size="2xl" as="h1" align="left">
+          {contentTitle}
+        </Title>
+        <Flex gap="2xl" direction="column" align={{sm: 'center', md: 'start'}}>
+          <Description variant="muted" size="lg" align="left">
+            {contentDescription}
+          </Description>
+          <Flex gap="sm">
+            <LinkButton
+              aria-label={t('Edit plan')}
+              to="/settings/billing/checkout/?referrer=checkout_success"
+            >
+              {t('Edit plan')}
+            </LinkButton>
+            <LinkButton
+              priority="primary"
+              aria-label={t('View your subscription')}
+              to={`/settings/billing/overview/${viewSubscriptionQueryParams}`}
+            >
+              {t('View your subscription')}
+            </LinkButton>
+          </Flex>
+        </Flex>
+      </Flex>
+      {isImmediateCharge ? (
+        <Receipt
+          {...commonChangesProps}
+          charges={invoice.charges}
+          planItem={planItem as InvoiceItem}
+          dateCreated={invoice.dateCreated}
+        />
+      ) : effectiveToday ? null : (
+        previewData &&
+        effectiveDate && (
+          <ScheduledChanges {...commonChangesProps} effectiveDate={effectiveDate} />
+        )
+      )}
+    </Flex>
+  );
+}
+
+export default CheckoutSuccess;
+
+const Title = styled(Heading)`
+  @media (max-width: ${p => p.theme.breakpoints.md}) {
+    text-align: center;
+  }
+`;
+
+const Description = styled(Text)`
+  @media (max-width: ${p => p.theme.breakpoints.md}) {
+    text-align: center;
+  }
+`;
+
+const StyledGrid = styled(Grid)`
+  & > :last-child {
+    justify-self: end;
+  }
+`;
+
+const ReceiptSlot = styled('div')`
+  width: 445px;
+  height: 7px;
+  border-radius: ${p => p.theme.borderRadius};
+  background: ${p => p.theme.gray200};
+  box-shadow: 0px 2px 4px 0px
+    ${p => Color(p.theme.black).lighten(0.08).alpha(0.15).toString()} inset;
+`;
+
+const ReceiptPaperContainer = styled('div')`
+  position: relative;
+  left: 62px;
+  top: -7px;
+  width: 320px;
+  overflow: hidden;
+`;
+
+const ReceiptPaperShadow = styled('div')`
+  position: relative;
+  z-index: 1000;
+  top: 0;
+  width: 320px;
+  height: 7px;
+  box-shadow: inset 0 10px 6px -6px
+    ${p => Color(p.theme.black).lighten(0.05).alpha(0.15).toString()};
+`;
+
+const ReceiptPaper = styled(Container)`
```

Comment:
Love that you have been using this, I just realized "none" should be a border option here too!